pkg/services/authn/authnimpl/service.go

```diff
@@ -320,6 +320,15 @@ func (s *Service) RegisterClient(c authn.Client) {
 	}
 }
 
+func (s *Service) IsClientEnabled(name string) bool {
+	client, ok := s.clients["auth.client."+name]
```

Comment:
Not sure abut this, the api should probably only accept valid auth client names. Most of then are already exposed https://github.com/grafana/grafana/blob/ad7f8042553ac1a260f49907e4a42ade8c5eb741/pkg/services/authn/authn.go#L20-L31 . Just add the missing one and don't alter the name here

Reply:
Yes! I changed this according to your comments